ethexe/processor/src/lib.rs

```diff
@@ -121,8 +109,7 @@ impl Processor {
         Ok(true)
     }
 
-    // TODO: deal with params on smart contract side.
-    pub fn handle_new_program(&mut self, program_id: ProgramId, code_id: CodeId) -> Result<H256> {
+    pub fn handle_new_program(&mut self, program_id: ProgramId, code_id: CodeId) -> Result<()> {
```

Comment:
mb check here that the program does not exist in db?

Reply:
program -> code_id matching does exactly the thing